src/main/java/org/kohsuke/github/GHRepositoryPublicKey.java

```diff
@@ -0,0 +1,46 @@
+package org.kohsuke.github;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import java.io.IOException;
+import java.net.URL;
+
+/**
+ * A public key for the given repository
+ *
+ * @author Aditya Bansal
+ */
+public class GHRepositoryPublicKey extends GHObject {
```

Comment:
nice! if you have a json response for this would be good to add a test so we can merge back into upstream

Reply:
done!!!!